src/WriterRegistrar.h

```diff
@@ -69,9 +69,10 @@ void clear();
 template <typename Module> class Registrar {
 public:
   /// \brief Register the writer module given in template parameter `Module`
-  /// under the
-  /// identifier `FlatbufferID`.
-  ///
+  /// under the identifier `FlatbufferID`.
+  /// ecdc-3447 this comment looks out of date, correct call is:
```

Comment:
Let's have a quick chat about this one.
I need a bit more context

Reply:
Looking back at the other branch it seems I made this comment early on in the process.  Looking now in context of more knowledge about the codebase and how the modules are registered this comment isn't valid, but I've expanded it to make the link with the other important parts of the stream registration more explicit.